mineinabyss-features/src/main/kotlin/com/mineinabyss/guilds/extensions/Guild.kt

```diff
@@ -17,27 +17,21 @@ import org.jetbrains.exposed.sql.*
 import org.jetbrains.exposed.sql.transactions.transaction
 
 fun Player.createGuild(guildName: String) {
-    val config = MIAConfig.data.guilds
+    val config = MIAConfig.data
     val newGuildName = guildName.replace("\\s".toRegex(), "") // replace space to avoid: exam ple
 
     newGuildName.broadcastVal()
-    if (newGuildName.length > config.maxLength) {
+    if (newGuildName.length > config.guildNameLength) {
         player?.error("Your guild name was longer than the maximum allowed length.")
-        player?.error("Please make it shorter than ${config.maxLength} characters.")
+        player?.error("Please make it shorter than ${config.guildNameLength} characters.")
         return
     }
 
-    if (!newGuildName.contains("^[a-zA-Z]*$".toRegex())) {
-        player?.error("Your guild name must be alpha-numerical.")
```

Comment:
its pretty redundant to say numbers are blocked even though the requirement is that its "alphanumerical", might probably be better to do like "must comprise of letters" instead of "must be alpha-numerical"

also im pretty certain 'alphanumerical' doesnt have hyphens